content/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -0,0 +1,269 @@
+
+CSRには、証明書の名前、ドメイン、IPが含まれますが、用途は指定されません。
+証明書を発行する際に、[正しい証明書の使用法](/docs/setup/best-practices/certificates/#all-certificates)を指定するのはCAの責任です。
+
+* `openssl`では、[`openssl ca`コマンド](https://superuser.com/questions/738612/openssl-ca-keyusage-extension)を使って行います。
+
+* `cfssl`では、[configファイルのusages](https://github.com/cloudflare/cfssl/blob/master/doc/cmd/cfssl.txt#L170)で指定します。
+
+お好みの方法で証明書に署名した後、証明書と秘密鍵をPKIディレクトリ(デフォルトでは`/etc/kubernetes/pki`)にコピーする必要があります。
+
+## 認証局(CA)のローテーション {#certificate-authority-rotation}
+
+Kubeadmは、CA証明書のローテーションや交換を最初からサポートしているわけではありません。
+
+CAの手動ローテーションや交換についての詳細は、[manual rotation of CA certificates](/docs/tasks/tls/manual-rotation-of-ca-certificates/)を参照してください。
+
+## 署名付きkubeletサービング証明書の有効化 {#kubelet-serving-certs}
+
+デフォルトでは、kubeadmによって展開されるkubeletサービング証明書は自己署名されています。
+これは、[metrics-server](https://github.com/kubernetes-sigs/metrics-server)のような外部サービスからキューブレットへの接続がTLSで保護されないことを意味します。
+新しいkubeadmクラスタ内のkubeletが適切に署名されたサービング証明書を取得するように設定するには、`kubeadm init`に以下の最小限の設定を渡す必要があります。
+
+```yaml
+apiVersion: kubeadm.k8s.io/v1beta3
+kind: ClusterConfiguration
+---
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+serverTLSBootstrap: true
+```
+
+すでにクラスタを作成している場合は、以下の手順で適応させる必要があります。
+
+ - kube-system` ネームスペースにある `kubelet-config-{{< skew latestVersion >}}` ConfigMap を見つけて編集します。
+
+そのConfigMapの`kubelet`キーの値として[KubeletConfiguration](/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration) ドキュメントを指定します。KubeletConfigurationドキュメントを編集し、`serverTLSBootstrap: true`を設定します。
+
+- 各ノードで、`/var/lib/kubelet/config.yaml`に`serverTLSBootstrap: true`フィールドを追加し、`systemctl restart kubelet`でkubeletを再起動します。
+
+`serverTLSBootstrap: true`フィールドは、kubeleサービングのブートストラップを有効にします。
+証明書を`certificates.k8s.io`APIにリクエストすることで、証明書を発行することができます。
+
+既知の制限事項として、これらの証明書のCSR(Certificate Signing Requests)はkube-controller-managerのデフォルトサイナーによって自動的に承認されないことがあります。
+[`kubernetes.io/kubelet-serving`](/docs/reference/access-authn-authz/certificate-signing-requests/#kubernetes-signers) を参照してください。
+
+これには、ユーザーまたはサードパーティのコントローラからのアクションが必要です。
+
+これらのCSRは、以下を使用して表示できます:
+
+```shell
+kubectl get csr
+NAME        AGE     SIGNERNAME                        REQUESTOR                      CONDITION
+csr-9wvgt   112s    kubernetes.io/kubelet-serving     system:node:worker-1           Pending
+csr-lz97v   1m58s   kubernetes.io/kubelet-serving     system:node:control-plane-1    Pending
+```
+
+承認するためには、次のようにします:
+```shell
+kubectl certificate approve <CSR-name>
+```
+
+デフォルトでは、これらのサービング証明書は1年後に失効します。
+
+Kubeadmは`KubeletConfiguration`フィールド`rotateCertificates`を`true`に設定します。これは有効期限が切れる間際に、サービング証明書のための新しいCSRセットを作成し、ローテーションを完了するために承認する必要があることを意味します。
+
+詳しくは[Certificate Rotation] (/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#certificate-rotation)をご覧ください。
+
+これらのCSRを自動的に承認するためのソリューションをお探しの場合は、以下をお勧めします。
+クラウドプロバイダーに連絡し、ノードの識別をアウトオブバンドのメカニズムで行うCSRの署名者がいるかどうか尋ねてください。
+
+{{% thirdparty-content %}}
```

Comment:
I checked [preview site for this PR](https://deploy-preview-31213--kubernetes-io-main-staging.netlify.app/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/).
The last section of [here](https://deploy-preview-31213--kubernetes-io-main-staging.netlify.app/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/#kubelet-serving-certs) has English text about "thirdparty-content" as is. (just before "Feedback" section.)
Could you localize those texts?
I think target file for those is [`data/i18n/ja/ja.toml`](https://github.com/kubernetes/website/blob/main/data/i18n/ja/ja.toml) and original English texts are on [`data/i18n/en/en.toml`](https://github.com/kubernetes/website/blob/main/data/i18n/en/en.toml#L234-L245).